### DISCUSSION TITLE
可以「複製本頁面網址」嗎

### DISCUSSION THREAD

**Discussion by KamiyaMinoru** [zimocode/smartup#322] [General]
不想要滑鼠移到網址列然後再用鍵盤ctrl+c複製貼上
有沒有手勢可以簡單複製正在看的網頁的網址?

**Comment by zimocode** [comment]
![image](https://user-images.githubusercontent.com/29518677/228266484-69fe01cf-4311-4c72-976e-c03559f7e6e0.png)

**Comment by KamiyaMinoru** [comment]
Thanks!!!
